schemas/com.snowplowanalytics.snowplow.badrows/loader_iglu_error/jsonschema/2-0-0

```diff
@@ -0,0 +1,657 @@
+                "properties": {
+                  "error": {
+                    "enum": ["ValidationError"]
+                  },
+                  "dataReports": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                      "properties": {
+                        "message": {
+                          "type": "string",
+                          "description": "Human-readable message describing the issue with the schema"
+                        },
+                        "path": {
+                          "type": ["string", "null"],
+                          "description": "JSON Path to an issue in the faulty JSON datum"
+                        },
+                        "keyword": {
+                          "type": ["string", "null"],
+                          "description": "JSON Schema Keywrod caused invalidation"
+                        },
+                        "targets": {
+                          "type": ["array", "null"]
+                        }
+                      },
+                      "required": [ "path", "message", "keyword", "targets" ],
+                      "additionalProperties": false
+                    }
+                  }
+                },
+                "required": [ "dataReports" ],
+                "additionalProperties": false
+              },
+              {
+                "description": "Schema is invalid and cannot be used to validate an instance",
+                "properties": {
+                  "error": {
+                    "enum": ["ValidationError"]
+                  },
+                  "schemaIssues": {
+                    "description": "List of problems in resolved JSON schema",
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "path": {
+                          "type": "string",
+                          "description": "JSON Path to an issue in the faulty JSON Schema"
+                        },
+                        "message": {
+                          "type": "string",
+                          "description": "Human-readable message describing the issue with the schema"
+                        }
+                      },
+                      "required": [ "path", "message" ],
+                      "additionalProperties": false
+                    }
+                  }
+                },
+                "required": [ "error" ],
+                "additionalProperties": false
+              }
+            ]
+          }
+        },
+        "required": ["schemaKey", "error"],
+        "additionalProperties": false
+      }
+    },
+    "payload": {
+      "type": "object",
+      "description": "The enriched event that resulted in a storage loader failure",
+      "properties": {
```

Comment:
Out of curiosity, is it not possible to specify that the value of an object should match another schema ? e.g. [atomic event](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/atomic/jsonschema/1-0-0) here

Reply:
No, unfortunately. It is often requested by users and would be a killer feature for a project like bad rows (we re-use a lot of schemas), but we never got there.